index.html

```diff
@@ -306,9 +306,38 @@ <h3>ratings</h3>
     <span class="iconify rating-fill" data-icon="mdi:star" data-inline="false"></span>
     <span class="iconify rating-fill" data-icon="mdi:star" data-inline="false"></span>
 </div>
-
     <span class="rating-num">5<span class="iconify" data-icon="mdi:star" data-inline="false"></span></span>
 
-</body>
+<!-- modal -->
+<h3>modal</h3>
```

Comment:
wrapping a particular section in a section tag will give more readability

Reply:
it's component site, will do in doc site